### PR TITLE
Cooler format

### PR DESCRIPTION
**Pull request by joachimwolff**
Python 3 tests should all pass now.

### CODE REVIEW

File: hicexplorer/HiCMatrix.py [hunk 1]
```diff
@@ -2014,5 +1942,6 @@ def toBytes(s):
     if isinstance(s, str):
         return bytes(s, 'ascii')
     if isinstance(s, list):
+        print("to bytes")
```
Comment (bgruening) [inline hicexplorer/HiCMatrix.py]: This should be removed as well, but we can do this in the main repo.

File: hicexplorer/hicPlotMatrix.py [hunk 2]
```diff
@@ -424,14 +424,16 @@ def getRegion(args, ma):
     chrom = region_start = region_end = idx1 = start_pos1 = chrom2 = region_start2 = region_end2 = idx2 = start_pos2 = None
     chrom, region_start, region_end = translate_region(args.region)
 
-    # print("len(cut_intervals)", len(ma.cut_intervals))
-    if type(next(iter(ma.interval_trees))) is np.bytes_:
+    if type(next(iter(ma.interval_trees))) is np.bytes_ or type(next(iter(ma.interval_trees))) is bytes:
```
Comment (bgruening) [inline hicexplorer/hicPlotMatrix.py]: `if type(next(iter(ma.interval_trees))) in [np.bytes_, bytes]`

Comment (joachimwolff) [reply hicexplorer/hicPlotMatrix.py]: Ah, I was searching for something like this. :) Thanks.

**Comment by bgruening** [comment]
Uh, a segmentation fault under OSX.

**Comment by fidelram** [comment]
Is this using the latest cooler format?

**Comment by joachimwolff** [comment]
It is now on the newest release of cooler, but the one bug is not fixed. It is not a big issue, I just want to check if it is a valid cooler format. The API gives us the function `cooler.io.is_cooler`, it returns `True` in Python 2 but `False` in Python 3. However if we ignore this, it works as it should be.

**Comment by bgruening** [comment]
restarted all builds

**Comment by bgruening** [comment]
All important tests are green! Liniting fails.
Liniting should happen in a first stage and the other tests should not start before.

Merging, and we need to fix this in the other branch.

**Pull request merged**